contrib/http_util/src/main/java/io/opencensus/contrib/http/AbstractHttpHandler.java

```diff
@@ -24,10 +24,18 @@
 import io.opencensus.trace.MessageEvent;
 import io.opencensus.trace.MessageEvent.Type;
 import io.opencensus.trace.Span;
+import io.opencensus.trace.Span.Options;
 import javax.annotation.Nullable;
 
 /** Base class for handling request on http client and server. */
 abstract class AbstractHttpHandler<Q, P> {
+  static final String HTTP_HOST = "http.host";
```

Comment:
Consider to have all these in a public file (constants) if people want to share the same values.

Reply:
sure.